dev/InfoBar/InfoBar.cpp

```diff
@@ -47,14 +48,19 @@ void InfoBar::OnApplyTemplate()
             const auto closeButtonName = ResourceAccessor::GetLocalizedStringResource(SR_InfoBarCloseButtonName);
             winrt::AutomationProperties::SetName(closeButton, closeButtonName);
         }
-
         // Setup the tooltip for the close button
         auto tooltip = winrt::ToolTip();
         const auto closeButtonTooltipText = ResourceAccessor::GetLocalizedStringResource(SR_InfoBarCloseButtonTooltip);
         tooltip.Content(box_value(closeButtonTooltipText));
         winrt::ToolTipService::SetToolTip(closeButton, tooltip);
     }
 
+    if (const auto iconTextblock = GetTemplateChildT<winrt::FrameworkElement>(c_iconTextBlockName, controlProtected))
```

Comment:
Are you able to set the icon in a way so that it is announced after the navigation landmark? That way the announcement when an action/close button is focused and InfoBar opened is "navigation landmark, icon, title, message" instead of "Icon, navigation landmark, title, message".

Reply:
@gabbybilka  I don't think we have control over this. We just provide the information and Narrator decides the order in which it is going to read.

Reply:
Ok, that's fine!